src/main/java/com/g4s8/ghman/web/GHAuthorizationTake.java

```diff
@@ -0,0 +1,62 @@
+package com.g4s8.ghman.web;
+
+import java.io.IOException;
+import javax.sql.DataSource;
+import com.g4s8.ghman.user.User;
+import com.g4s8.ghman.user.Users;
+import com.jcabi.http.request.JdkRequest;
+import com.jcabi.http.response.JsonResponse;
+import com.jcabi.log.Logger;
+import com.jcabi.http.Request;
+import org.takes.Response;
+import org.takes.Take;
+import org.takes.facets.auth.RqAuth;
+import org.takes.rq.RqHref;
+import org.takes.rs.RsText;
+
+/**
+ * Github Authorization Take.
+ * @since 1.0
+ * @checkstyle LineLengthCheck (500 lines)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
+ * @checkstyle ClassFanOutComplexityCheck (500 lines)
+ */
+final class GHAuthorizationTake implements Take {
```

Comment:
@samihibrahim this should be named following the pattern `TkXXXX` since it is an implementation of `Take` (the same as other implementations in the project).